config/core/build-configs.yaml

```diff
@@ -251,6 +251,29 @@ fragments:
       - 'CONFIG_MMC_SDHCI=y'
       - 'CONFIG_MMC_SDHCI_PCI=y'
       - 'CONFIG_MMC_SDHCI_ACPI=y'
+      - 'CONFIG_CHROME_PLATFORMS=y'
+      - 'CONFIG_CHROMEOS_LAPTOP=y'
+      - 'CONFIG_CHROMEOS_TBMC=y'
+      - 'CONFIG_DM_CRYPT=y'
+      - 'CONFIG_DM_THIN_PROVISIONING=y'
+      - 'CONFIG_DM_INIT=y'
+      - 'CONFIG_DM_VERITY=y'
+      - 'CONFIG_CFS_BANDWIDTH=y'
+      - 'CONFIG_USER_NS=y'
+      - 'CONFIG_BLK_DEV_NVME=y'
+      - 'CONFIG_CROS_EC=y'
+      - 'CONFIG_CROS_EC_LPC=y'
+      - 'CONFIG_CROS_EC_I2C=y'
+      - 'CONFIG_IIO=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS_CORE=m'
+      - 'CONFIG_IIO_CROS_EC_SENSORS=m'
+      - 'CONFIG_MFD_CROS_EC_DEV=m'
```

Comment:
These extend the cros-ec-tests coverage, from not testing the cros-ec to test basic functionalities of the cros-ec. I ran it on the three devices (grunt, coral, octopus)

Without this fragment:
* https://lava.collabora.co.uk/results/4247977
* https://lava.collabora.co.uk/results/4264092
* https://lava.collabora.co.uk/results/4267113

With the fragment, the results are:
* https://lava.collabora.co.uk/results/4276864
* https://lava.collabora.co.uk/results/4276865
* https://lava.collabora.co.uk/results/4276866

Reply:
Great, so I think we should keep these in this PR then.